lib/baseConfig.ts

```diff
@@ -49,6 +49,15 @@ export interface BaseOptions {
 	 * @default 'dist/vendor.LICENSE.txt'
 	 */
 	thirdPartyLicense?: false | string
+	/**
+	 * Customize the asset file names.
+	 * Similar to `output.assetFileNames` in rollup config,
+	 * but if returns undefined, then this config defaults is be used.
+	 *
+	 * @example Move CSS styles to `styles/style.css` instead of the default `css/[entrypoint-name].css`:
```

Comment:
I think this is not a good example as you will have to set `cssCodeSplit` to false. Otherwise the styles would override each other.

Reply:
The example was based on `nextcloud/password-confirmation`, where currently styles are renamed manually by adding `&& mv ./dist/main.css ./dist/style.css` (only ti production build, dev build just doesn't work).

There is no `cssCodeSplit` and this custom function works.

But there is only one entrypoint